src/main/java/dk/cngroup/lentils/controller/ClientController.java

```diff
@@ -91,6 +93,17 @@ public String verifyCodeword(@PathVariable("id") final Long id,
                                  final BindingResult result,
                                  final Model model) {
         Cypher cypher = cypherService.getCypher(id);
+        String status = statusService.getStatusNameByTeamAndCypher(user.getTeam(), cypher);
+
+        final FinalPlace finalPlace = finalPlaceService.getFinalPlace();
+        if (finalPlace.getOpeningTime() == null || finalPlace.getOpeningTime().isBefore(LocalDateTime.now())) {
+            FieldError error = new FieldError("codeword", "guess", "Hra již byla ukončena");
```

Comment:
OPTIONAL: Uz tady mame hodne tech magic strings a rikam si, ze zrovna tady bych to extraktoval do konstant. Z tech dvou prvnich parametru "codeword" a "guess" nevim, k cemu slouzi. Jinak mozna by se hodilo extrahovat i ty message z tech exceptions..

Reply:
To ještě změním, jsem pro.